BaseTools/Source/C/Common/PcdValueCommon.h

```diff
@@ -18,12 +18,6 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #define __FLEXIBLE_SIZE(Size, TYPE, Field, MaxIndex)   if (__FIELD_SIZE(TYPE, Field) == 0) Size = MAX((__OFFSET_OF(TYPE, Field) + __ARRAY_ELEMENT_SIZE(TYPE, Field) * (MaxIndex)), Size)
 #define __ARRAY_SIZE(Array) (sizeof(Array)/sizeof(Array[0]))
 
-#if defined(_MSC_EXTENSIONS)

```

Comment:
The changes in this commit should be "MU_CHANGES" for now.

Reply:
To be clear here, comment out and tag as a MU_CHANGE.

Reply:
Done.